Gems/LyShine/Code/Source/RenderGraph.cpp

```diff
@@ -509,6 +509,10 @@ namespace LyShine
             if (!m_dynamicDraw)
             {
                 m_dynamicDraw = uiRenderer->CreateDynamicDrawContextForRTT(GetRenderTargetName());
+                if (m_dynamicDraw)
+                {
+                    m_dynamicDraw->SetViewport(AZ::RHI::Viewport(m_viewportX, m_viewportWidth, m_viewportY, m_viewportHeight));
```

Comment:
You may need to up set scissor too if the viewport size is larger than window size.